capstone/src/main/java/com/google/sps/servlets/FileDisplayServlet.java

```diff
@@ -0,0 +1,78 @@
+package com.google.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.MultipartConfig;
+import java.io.PrintWriter;
+import com.google.gson.Gson; //Convert json to string
+
+
+
+@WebServlet("/display")
+@MultipartConfig
+public class FileDisplayServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  
+    // String userId = request.getParamter("userId");
```

Comment:
Is this going to change to something in the future? Consider adding a TODO / Github issue reference here!

Reply:
Yes, it is going to later. I discussed this with him on Friday.

Reply:
Nice! These TODOs will make it easier to track work that's promised but needs to wait.